src/test/java/org/fundacionjala/pivotal/steps/EditProjectSteps.java

```diff
@@ -20,7 +20,7 @@
 
     private DashBoard dashBoard = new DashBoard();
 
-    private Map<DescriptionValues, Object> descriptions;
+    private Map<DescriptionValues, String> descriptions;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "descriptions" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)